frontend/simple/views/Invite.vue

```diff
@@ -98,22 +103,29 @@ export default {
       this.members.splice(index, 1)
     },
     submit: async function () {
-      for (let i = 0; i < this.members.length; i++) {
-        let member = this.members[i]
-        let events = await backend.eventsSince(member.contractId, member.contractId)
-        let [contract, ...actions] = events.map(e => {
-          return Events[e.entry.type].fromObject(e.entry, e.hash)
-        })
-        let state = contract.toVuexState()
-        actions.forEach(action => {
-          let type = action.constructor.name
-          contract.constructor.vuex.mutations[type](state, action.data)
-        })
-        let mailbox = await backend.latestHash(state.attributes.mailbox)
-        let invite = new Events.PostInvite({groupId: this.$store.state.currentGroupId}, mailbox)
-        await backend.publishLogEntry(state.attributes.mailbox, invite)
+      try {
+        this.errorMsg = null
+        // TODO as members are successfully invited display in a seperate invitees grid and add them to some validation for duplicate invites
+        for (let i = 0; i < this.members.length; i++) {
+          let member = this.members[i]
+          // We need to have the latest mailbox attribute for the user
+          let state = await latestContractState(member.contractId)
+          let mailbox = await backend.latestHash(state.attributes.mailbox)
+          let date = new Date()
+          // We need to post the invite to the users' mailbox contract
+          let invite = new Events.PostInvite({groupId: this.$store.state.currentGroupId, sentDate: date.toString()}, mailbox)
+          await backend.publishLogEntry(state.attributes.mailbox, invite)
+          // We need to make a record of the invitation in the group's contract
```

Comment:
Nice comments, this makes much more sense now. For the future though, please put a colon after `// TODO`, primarily to maintain consistency with the other `// TODO:`s in the source, but also because I think some plugins depend on that to do TODO management.